src/header/AddInput.js

```diff
@@ -0,0 +1,15 @@
+import LINKIMAGE from '../img/link.svg';
+
+export default function AddInput() {
+  return (
+    <div className="AddInput">
+      <div className="AddInput-input">
+        <div className="AddInput-fnc">
+          <img src={LINKIMAGE} alt="" />
+          <input type="text" placeholder="링크를 추가해 보세요"/> 
+          <button><div className="text">추가하기</div></button>
```

Comment:
```suggestion
          <button><span className="text">추가하기</span></button>
```
[중요도: 중간] 개인적으로 텍스트 관련에 스타일을 먹일 때는 span, p같은 것을 사용하는 것을 추천드립니다~

Reply:
88cf1e9acb5fa37f76130100f71dd2c23aa78440 
넵 참고하겠습니다 수정 완료했습니다